crates/bevy_time/src/lib.rs

```diff
@@ -69,23 +71,27 @@ impl Plugin for TimePlugin {
 /// you may prefer to set the next [`Time`] value manually.
 #[derive(Resource, Default)]
 pub enum TimeUpdateStrategy {
+    /// [`Time`] will be automatically updated according to the length of the previous frame.
     #[default]
     Automatic,
-    // Update [`Time`] with an exact `Instant` value
+    /// [`Time`] will be updated to the specified [`Instant`] value.
```

Comment:
```suggestion
    /// [`Time`] will be updated to the specified [`Instant`] value each frame.
```

(If I understood correctly)